packages/api-sdk/package.json

```diff
@@ -27,7 +27,9 @@
     "lint": "eslint --cache --max-warnings=0 src",
     "build:mcp": "bun src/mcp-server/build.mts",
     "build": "npm run build:mcp && tshy",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "format:check": "prettier --check . --ignore-path=../../.prettierignore",
```

Comment:
we had to add a post-processing script for `package.json` to make this possible...we'll might need something similar for the package publishing workflows.